config/environments/test.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.configure do
+
```

Comment:
This line is not needed.
You might want to add the following rule:
```
Layout/EmptyLinesAroundBlockBody:
  Description: checks if empty lines around the bodies of blocks
  Enabled: no_empty_lines
```

Which looking at the documentation should be the default: https://rubocop.readthedocs.io/en/latest/cops_layout/#layoutemptylinesaroundblockbody